client/components/Entry/CreateEntryForm.vue

```diff
@@ -5,16 +5,22 @@ import BlockForm from '@/components/common/BlockForm.vue';
 export default {
   name: 'CreateEntryForm',
   mixins: [BlockForm],
+  props: {
+      type: {
+          type: String,
+          required: false
```

Comment:
this one could probably be true?

Reply:
Hmm, since the edit form uses the create form as a mixin, apparently requiring props here will introduce a Vue warning in the console for the edit form, so I figured it's fine to just not require it...